views/components/Sidebar/NavItem.jsx

```diff
@@ -5,10 +5,16 @@ import SidebarLink from './SidebarLink'
 const NavItem = ({ to, Icon, label }) => {
   return (
     <ListItem component={SidebarLink} to={to}>
-      <ListItemIcon sx={{ minWidth: 40, alignSelf: 'center' }}>
-        <Icon />
+      <ListItemIcon
+        sx={{ minWidth: 20, width: 20, height: 20, alignSelf: 'center' }}
+      >
+        <Icon sx={{ height: 20, width: 20 }} />
       </ListItemIcon>
-      <ListItemText primary={label} sx={{ color: 'text.primary' }} />
+      <ListItemText
+        primary={label}
+        sx={{ color: 'text.primary' }}
+        primaryTypographyProps={{ sx: { fontWeight: 500 } }}
+      />
```

Comment:
```suggestion
        sx={{ color: 'text.primary', fontWeight: 500}}
      />
```

I am conflicted about using the primaryTypographyProps here we can get the same result from the sx prop as it should apply to the children.

Reply:
unfortunately the prop was not being applied to the child elements which is why I did it this way.